devel/delete-from-ghcr.js

```diff
@@ -0,0 +1,57 @@
+// Delete Nextstrain images with a specific tag from the GitHub Container registry.
+//
+// This file is intended for use by the GitHub action "actions/github-script".
```

Comment:
I guess using `actions/github-script` makes it harder to test/perform locally. Ah well.

Reply:
[This force-push](https://github.com/nextstrain/docker-base/compare/c2d069f17aaa0a830bb02ea93fb80cfdf1d72970..68d46254213d48c8d93b8b2032527729e682ea0a) adds a note on how to use it locally if you wish. Note that this whole pushing intermediates to ghcr.io thing is CI-specific and not used during local dev – that uses a local registry which can be cleaned up using `devel/stop-localhost-registry`.